src/API/AbstractPluginActions.php

```diff
@@ -142,7 +142,7 @@ public function patchPluginSettings()
 
         $body = $this->request->getBody();
         $value = $body['value'];
-        $options = $this->dataStore->set($settingId, $this->api->createPluginSettingObject($settingId, $value, true, null));
+        $options = $this->dataStore->set($settingId, $this->api->createPluginSettingObject($settingId, $value, true, true));
```

Comment:
Same problem as [here](https://github.com/cloudflare/cloudflare-plugin-backend/pull/19) the last argument doesn't matter.